bin/cirrus_promote_maven

```diff
@@ -7,9 +7,14 @@ set -euo pipefail
 source cirrus-env PROMOTE
 source set_maven_build_version $BUILD_NUMBER
 curl -sfSL -H "Authorization: Bearer $GCF_ACCESS_TOKEN" "$PROMOTE_URL/$GITHUB_REPO/$GITHUB_BRANCH/$BUILD_NUMBER/$PULL_REQUEST"
+
+#Add github commit status with build number
+BUILD_URL="${ARTIFACTORY_URL}/webapp/#/builds/sonar-dummy/${BUILD_NUMBER}/"
```

Comment:
`sonar-dummy` should be replaced with the actual project name.

Reply:
oups indeed, thanks